src/views/bing.py

```diff
@@ -14,7 +14,13 @@ def __init__(self, interface_layer):
             ("/i(mage)?\s(?P<term>[^$]+)$", self.bing_image_search)
         ]
 
-    def bing_image_search(self, message, match):
+    def bing_image_search(self, message, match, thumbnail=None):
         req = requests.get("https://api.datamarket.azure.com/Bing/Search/v1/Image?Query=%27{}%27&$format=json&$top=1".format(match.group("term")), auth=("",config.bing_api))
-        image_url = urllib.unquote(req.json()['d']['results'][0]['MediaUrl'].encode('utf-8'))
-        self.image_sender.send_by_url(jid=message.getFrom(), file_url=image_url)
+
+        if thumbnail is not None:
+            image_url = urllib.unquote(req.json()['d']['results']['Thumbnail']['MediaUrl'].encode('utf-8'))
+            self.image_sender.send_by_url(jid=message.getFrom(), file_url=image_url, ftype=".jpg")
```

Comment:
To not modified "send_by_url" you can append ftype in this line:

```
self.image_sender.send_by_url(jid=message.getFrom(), file_url=image_url+".jpg")
```
Unless the ftype change in the future.